.github/workflows/check_release_tag.py

```diff
@@ -1,16 +1,32 @@
 # -*- coding: utf-8 -*-
 """Check that the GitHub release tag matches the package version."""
 import argparse
-import json
+import ast
+from pathlib import Path
+
+
+def get_version_from_module(content: str) -> str:
+    """Get the __version__ value from a module."""
+    # adapted from setuptools/config.py
+    try:
+        module = ast.parse(content)
+    except SyntaxError as exc:
+        raise IOError(f'Unable to parse module: {exc}')
+    try:
+        return next(
+            ast.literal_eval(statement.value) for statement in module.body if isinstance(statement, ast.Assign)
+            for target in statement.targets if isinstance(target, ast.Name) and target.id == '__version__'
+        )
+    except StopIteration:
+        raise IOError('Unable to find __version__ in module')
```

Comment:
Since we have full control over the `aiida/__init__.py` file, is it really necessary to add this function here?

Reply:
How else would you propose to retrieve the source version?

Reply:
Yeh but look how beautiful this code is lol